README.md

```diff
@@ -73,6 +75,20 @@ This will leave the binaries in the `bin/` subdirectory; you may use thes direct
 Consult the Nesdev Wiki [Installing CC65][nd-cc65] page for some hints, including Windows installs. However, the Debian packages they suggest from [trikaliotis.net] appear to have signature errors.
 
 
+### Building/Installing lzsa
+
+#### Linux Builds from Source
+
+The `lzsa` compression utility is used for some resources packaged into the ROM, and is available [on Github](https://github.com/emmanuel-marty/lzsa); clone and build it with:
+
+	git clone git@github.com:emmanuel-marty/lzsa.git
+	make
```

Comment:
we could perhaps mention that replacing clang by gcc in the makefile will enable people that only have gcc installed to compile it as well without further hassle?